lib/driver.js

```diff
@@ -406,6 +406,15 @@ export class XCUITestDriver extends BaseDriver {
     return !(CAP_NAMES_NO_XCODEBUILD_REQUIRED.some((x) => Boolean(this.opts[x])));
   }
 
+  /**
+   * Print deprecated capabilities.
+   */
+  printDeprecatedCaps() {
+    if (!_.isUndefined(this.opts.useSimpleBuildTest)) {
+      this.log.warn("The capability 'appium:useSimpleBuildTest' is deprecated and will be removed.");
```

Comment:
do we still need this message if `deprecated` above is set true?

Reply:
perhaps it would be just enough to rely on the base driver behaviour. I assume it should be aligned with other drivers

Reply:
initially though to add more context why, but yes, could be removed. Actually older Appium may show deprecation message by basedriver for non-usage thus I thought this also helped. But I can remove them.